### ISSUE TITLE
Missing escape of single quote. PHP Parse error produced.

### ISSUE THREAD

**Issue by dfelton** [tzyganu/UMC1.9#105]
When the "Note" attribute of an entity's field/attribute contains single quote(s), they are not being escaped when put into the corresponding instance of the Varien_Data_Form_Element_Abstract element for the field/attribute. This results in an improperly ended string, producing a PHP Parse error.  

![img1](https://cloud.githubusercontent.com/assets/2659558/10462427/4bdf3aca-71a4-11e5-84aa-54c3a79686a1.png)

`app/code/{{codepool}}/{{Namespace}}/{{Module}}/Block/Adminhtml/{{Entity}}/Edit/Form.php`
Example Generated Source:

```php
$fieldset->addField(
    'foo_bar',
    'text',
    array(
        'label' => Mage::helper('foo_bar')->__('Foo Bar'),
        'name'  => 'foo_bar',
        'note'	=> $this->__('Foo's Bar'),
        'required'  => true,
        'class' => 'required-entry',
    )
);
```

**Comment by tzyganu** [comment]
Thanks for spotting this. I will fix it as soon as I have the time. Of course your PR is always welcomed :smile:

**Comment by dfelton** [comment]
@tzyganu your welcome. Yes if I have time first I'll dig into it as well. Oh how I wish there was 48 hours in a day, and the human body was built to handle such hours.

**Comment by fffab** [comment]
Hi, @tzyganu @dfelton i did just spotted the same on 'label'. For example
'label' => Mage::helper('foo_bar')->__('Foo's Bar'),

**Comment by tzyganu** [comment]
@dfelton @fffab These issues should be fixed in version 1.9.6.0
